src/oatpp/web/client/ApiClient.hpp

```diff
@@ -389,10 +389,10 @@ template<>
 struct ApiClient::TypeInterpretation<oatpp::Boolean> {
   static oatpp::String toString(const oatpp::String &typeName, const oatpp::Boolean &parameter) {
     (void) typeName;
-    if (parameter) {
-      return utils::conversion::boolToStr(*parameter);
+    if(parameter.get() != nullptr) {
+        return utils::conversion::boolToStr(*parameter);
     }
-    return nullptr;
+    return oatpp::String("false", 5, false);
```

Comment:
I believe it's better to return `nullptr` if `nullptr` was passed.
I agree that the silent crash is not good, but as for this exact pice of functionality, returning `nullptr`
is expected behavior.